app/cmd/start.go

```diff
@@ -146,89 +154,176 @@ func start(c *cli.Context) (err error) {
 		return err
 	}
 
+	var wg sync.WaitGroup
+
+	waitCount := 4
+	if spdkEnabled {
+		waitCount++
+	}
+	wg.Add(waitCount)
```

Comment:
Suggest removing this, but execute `wg.Add(1)` when needed later.

Reply:
Remove WaitGroup. Use errgroup instead.